Makefile

```diff
@@ -141,7 +143,7 @@ $(VENDOR_DIR):
 
 $(BUILD_DIR)/server: $(BUILD_DIR) $(VENDOR_DIR)
 	$(DOCKER_EXEC) bash -xc '$(DOCKER_DEPS) \
-		CGO_ENABLED=0 go build --ldflags "-s -w" \
+		CGO_ENABLED=0 go build --ldflags "-s -w -X main.version=${VERSION}" \
```

Comment:
Just information, stored into binary file. IT is very helpful while deployment and testing